### PR TITLE
Tests of Spice listening sockets

### PR DESCRIPTION
**Pull request by rdkdd**
172916 & 172917 implementation is dependent on my previous PR to avocado-vt (https://github.com/avocado-framework/avocado-vt/pull/855) and tp-spice (https://github.com/spiceqa/tp-spice/pull/51)

Signed-off-by: Radek Duda <rduda@redhat.com>

### CODE REVIEW

**Comment by Andrei-Stepanov** [comment]
I sent an email to avocado-devel mail listing asking them to accept your pull req.
Until it is accepted it is better not to merge this pullreq.

**Comment by rdkdd** [comment]
ok thanks

File: spice/cfg/tests-variants.cfg [hunk 1]
```diff
@@ -438,6 +443,79 @@ variants:
...
+                - 172920:
+                    spice_addr = blabla
+                    start_vm = no
+
+                - 172921:
+                    spice_addr = 127.0.0.256
+                    start_vm = no
+
+                - 172922:
+                    spice_addr = 127.0.0.0.1
```
Comment (Andrei-Stepanov) [inline spice/cfg/tests-variants.cfg]: Please confirm that "127.0.0.0.1" is correct value.

Comment (rdkdd) [inline spice/cfg/tests-variants.cfg]: yes it is correct. It is negative TC.  qemu-kvm should exit with Error

**Pull request closed without merging**